webapp/resources/templates/fragments/pwdupdateform.html

```diff
@@ -7,7 +7,12 @@
 </script>
 
 <div class="alert alert-info">
-    <h2 th:utext="#{screen.mustchangepass.heading}"/>
+    <div th:if="${execInfo.processedTemplateName} == 'casExpiredPassView'">
```

Comment:
I think that there is no such flag (or at least I can't think of any), but again I might be wrong. Perhaps @mmoayyed could shed some light on this.